ckanext/datagovuk/helpers.py

```diff
@@ -297,3 +297,8 @@ def roles():
 
 def google_analytics_tracking_id():
     return config.get('ckan.google_analytics_tracking_id')
+
+def is_central_gov_organogram(pkg_name):
+    pkg_dict = Package.by_name(pkg_name).as_dict()
+    pkg_extras = pkg_dict.get('extras')
+    return 'schema-vocabulary' in pkg_extras and pkg_extras['schema-vocabulary'] == 'd3c0b23f-6979-45e4-88ed-d2ab59b005d0'
```

Comment:
We could simplify this to `pkg_extras.get('schema-vocabulary') == 'd3c0b23f-6979-45e4-88ed-d2ab59b005d0'`.

Reply:
True, I just copied the pattern from other code